.circleci/config.yml

```diff
@@ -35,7 +35,12 @@ jobs:
       # Run tests
       - run: yarn lint
       - run: yarn build-noemit
-      - run: yarn test
+      - run:
+          name: Run tests
+          command: yarn test
+          environment:
+            PRE_SHARED_KEY: "12345"
+            ALLOW_NOTIFY_IP_SOURCE_RANGE: "::ffff:ac13:1/112"
```

Comment:
i suspect the current regex support only ipv4 cidrs so I'd replace this one with 172.19.3.0/112

Reply:
The regex is: https://github.com/teamdigitale/italia-backend/pull/68/files#diff-d5b02c5e87ab6689ba71e2e5207b7120R142
is supports both ipv4 and ipv6

Reply:
the question is: what kind of address do we get from the load balancer? have you checked already?

Reply:
No but this code correctly handle both ipv4 and ipv6 addresses